tests/integration/multi_cluster_deploy_test.go

```diff
@@ -113,39 +120,65 @@ func (mrc *MultiClusterDeploySuite) startCluster(namespace, store string, errCh
 }
 
 func (mrc *MultiClusterDeploySuite) TearDownSuite() {
-	if mrc.T().Failed() {
-		gatherAllRookLogs(mrc.k8sh, mrc.Suite, mrc.installer.Env.HostType, installer.SystemNamespace(mrc.namespace1), mrc.namespace1)
-		gatherAllRookLogs(mrc.k8sh, mrc.Suite, mrc.installer.Env.HostType, installer.SystemNamespace(mrc.namespace1), mrc.namespace2)
+	mrc.o.TearDown()
+}
+
+//BaseTestOperations struct for handling panic and test suite tear down
+type MCTestOperations struct {
+	installer   *installer.InstallHelper
+	installData *installer.InstallData
+	k8sh        *utils.K8sHelper
+	T           func() *testing.T
+	namespace1  string
+	namespace2  string
+}
+
+//GetBaseTestOperations creates new instance of BaseTestOperations struct
```

Comment:
cut and paste comment has wrong func name

Reply:
done.